flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/EmbeddedRocksDBStateBackendTest.java

```diff
@@ -145,6 +149,7 @@ public static List<Object[]> modes() {
     private RocksDB db = null;
     private ColumnFamilyHandle defaultCFHandle = null;
     private RocksDBStateUploader rocksDBStateUploader = null;
+    private int maxParallelism = 2;
```

Comment:
I think making `maxParallelism` as a private field is a bit strange. We can just pass `2` to the `RocksDBTestUtils.builderForTestDB` method.